bin/tools/tools_common.ml

```diff
@@ -134,3 +137,40 @@ let exec_command dev_tool =
   in
   Cmd.v info term
 ;;
+
+let env_command =
+  let term =
+    let+ builder = Common.Builder.term
+    and+ fish =
+      Arg.(
+        value
+        & flag
+        & info [ "fish" ] ~doc:"Print command for the fish shell raher than posix shells")
+    in
+    let _ = Common.init builder in
```

Comment:
I think it would be good to ascribe the result type here, to make sure that `_` does not get bound to a curried function application if `Common.init` gets extended with an additional argument.